app/src/main/java/com/github/se/gatherspot/ui/EventUI.kt

```diff
@@ -0,0 +1,236 @@
+
+              Text(
+                  text = "Number of attendees",
+                  modifier =
+                      Modifier.align(Alignment.CenterHorizontally).testTag("attendeesInfoTitle"),
+                  fontWeight = FontWeight.Bold)
+
+              // Event Capacity
+              Row(modifier = Modifier.testTag("attendeesInfo")) {
+                Text("Min: ", fontWeight = FontWeight.Bold)
+                Text("${event.attendanceMinCapacity}")
+                Spacer(modifier = Modifier.width(100.dp))
+                Text("Current: ", fontWeight = FontWeight.Bold)
+                Text(text = "${event.registeredUsers?.size ?: 0}")
+                Spacer(modifier = Modifier.width(100.dp))
+                Text("Max: ", fontWeight = FontWeight.Bold)
+                Text(text = "${event.attendanceMaxCapacity}")
+              }
+
+              // Categories
+              Spacer(modifier = Modifier.height(16.dp))
+              FlowRow(modifier = Modifier.testTag("categoriesRow")) {
+                event.categories?.forEach { interest -> Chip(interest = interest) }
+              }
+
+              // Map View Placeholder
+              Spacer(modifier = Modifier.height(16.dp))
+              // TODO: Implement the actual map and test it
+              Box(
+                  modifier =
+                      Modifier.height(200.dp)
+                          .fillMaxWidth()
+                          .background(Color.Gray)
+                          .testTag("mapView")) {
+                    // Here should be the code to integrate the actual map
+                    BasicText(text = "Map Placeholder")
+                  }
+              // Event Dates and Times
+              Spacer(modifier = Modifier.height(16.dp))
+              Row(modifier = Modifier.testTag("eventDatesTimes")) {
+                Column {
+                  Text("Event Start:", fontWeight = FontWeight.Bold)
+                  event.eventStartDate?.let { startDate ->
+                    Text(startDate.format(DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM)))
+                  }
+                  event.timeBeginning?.let { startTime ->
+                    Text(startTime.format(DateTimeFormatter.ofLocalizedTime(FormatStyle.SHORT)))
+                  }
+                }
+                Spacer(modifier = Modifier.width(16.dp))
+                Column {
+                  Text("Event End:", fontWeight = FontWeight.Bold)
+                  event.eventEndDate?.let { endDate ->
+                    Text(endDate.format(DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM)))
+                  }
+                  event.timeEnding?.let { endTime ->
+                    Text(endTime.format(DateTimeFormatter.ofLocalizedTime(FormatStyle.SHORT)))
+                  }
+                }
+              }
+
+              // Inscription Limit Date and Time
+              Spacer(modifier = Modifier.height(8.dp))
+              Text(
```

Comment:
Inscription date and time limits are optionals. What if there are not specified ?

Reply:
The question mark after the inscription date shows that the text will not be showed if the date is null.